time_monkey/settings/base.py

```diff
@@ -0,0 +1,119 @@
+"""
+Django settings for time_monkey project.
+
+Generated by 'django-admin startproject' using Django 1.11.12.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
```

Comment:
You should not delete `PRIV_KEY` at all from here. Add here, but as a comment and add description what is it used for.